include/package.mk

```diff
@@ -258,7 +259,7 @@ define Build/CoreTargets
 
   ifneq ($(CONFIG_AUTOREMOVE),)
     compile:
-		-touch -r $(PKG_BUILD_DIR)/.built $(PKG_BUILD_DIR)/.autoremove 2>/dev/null >/dev/null
+		touch -r $(STAMP_BUILT) $(STAMP_REMOVED) 2>/dev/null >/dev/null
```

Comment:
If you blame 6-7 years ago also this was added to the non fatal function. Wonder if we should keep this. I guess if STAMP_BUILD doesn't exist this will fail.

Reply:
I cannot imagine it failing in good circumstances, since it would now use the variable for the stampfiles.

the "built" stampfile must exist before this is ran. An error would suggest that someone who is editing the makefile made some mistake where autoremove happens before the built stampfile is made.

maybe the `2>/dev/null` can be removed

Reply:
I ended up making STAMP_REMOVED not rely on STAMP_BUILT at all

removal causes timestamp of the build directory to change, it would be useful to have a stampfile both before and after the timestamp of the directory